minitask2/akhmad/behaviour.py

```diff
@@ -0,0 +1,196 @@
+#!usr/bin/env python
+
+import rospy
+from tf.transformations import euler_from_quaternion
+
+from sensor_msgs.msg import LaserScan
+from geometry_msgs.msg import Twist
+from nav_msgs.msg import Odometry
+
+from math import radians, sqrt
+import random
+
+move_cmd = Twist()
```

Comment:
I don't think we should be using global variables here, why not just use member variables?

Reply:
Do you mean to use them as instance variables?

Reply:
Yeah, like `self.move_cmd`